src/azure-cli-core/azure/cli/core/commands/validators.py

```diff
@@ -119,3 +119,11 @@ def _dest_to_option(dest):
             forbidden_string = ', '.join(_dest_to_option(x) for x in included_forbidden)
             error = '{}\n\tnot applicable: {}'.format(error, forbidden_string)
         raise CLIError(error)
+
+
+def validate_edge_zone(cmd, namespace):  # pylint: disable=unused-argument
```

Comment:
I don't think we need to add such global validation here because it is easy to pass edge zone information in your command code as shown in https://github.com/Azure/azure-cli/blob/50f06e3e35f8c720597358188cae9f38fc179d98/src/azure-cli/azure/cli/command_modules/storage/operations/account.py#L160 and the exposed parameter is extended location not edge zone.

Reply:
OK. I removed it from core.